egs/wsj/s5/steps/nnet3/lstm/train.sh

```diff
@@ -103,7 +103,6 @@ egs_opts=
 transform_dir=     # If supplied, this dir used instead of alidir to find transforms.
 cmvn_opts=  # will be passed to get_lda.sh and get_egs.sh, if supplied.
```

Comment:
No need to change this script, it is deprecated so I don't want to waste time testing the change.